polyglotdb/sql/models.py

```diff
@@ -150,6 +150,28 @@ class SpeakerProperty(Base):
     value = Column(String(250), nullable = False)
 
 
+class SpeakerAnnotation(Base):
+    __tablename__ = 'speaker_annotation'
+
+    #speaker id
+    id = Column(Integer, primary_key = True)
+
+    speaker_id = Column(Integer, ForeignKey('speaker.id'), nullable=False)
+
+    annotation_id = Column(Integer, ForeignKey('annotation_type.id') , nullable=False)
+
+    property_type_id = Column(Integer, ForeignKey('property_type.id'), nullable=False)
+
+    property_type = relationship(PropertyType)
+
+    annotation_type = relationship(AnnotationType)
+
+    speaker = relationship(Speaker)
+
+
+
+    numerical_value = Column(Integer)
```

Comment:
probably want:
```
value = Column(Float, nullable = False)
```

to be consistent with the NumericProperty class.  Also, if it's an integer, it'll truncate information.